### DISCUSSION TITLE
Option to select which feeds user can view

### DISCUSSION THREAD

**Discussion by huhas12** [FreshRSS/FreshRSS#4925] [Ideas]
Hello, 

Is it possible to restict user access/view for selected feeds or to select the feeds can a user see?

**Comment by math-GH** [comment]
Could you please explain your use case?

**Reply by huhas12** [reply]
Hello math-GH, 

Wondering if admin can assign specific categories of rss feeds to users individually and restict users on registering a new rss feed unless the admin gives permission.

**Reply by math-GH** [reply]
There is a extension for white listening: https://github.com/aledeg/xExtension-WhiteList
Maybe it is useful for you?
